plugins/flytekit-data-fsspec/flytekitplugins/fsspec/persist.py

```diff
@@ -115,7 +115,11 @@ def put(self, from_path: str, to_path: str, recursive: bool = False):
             from fsspec.utils import other_paths
 
             lfs = LocalFileSystem()
-            lpaths = lfs.expand_path(from_path, recursive=recursive)
+            try:
+                lpaths = lfs.expand_path(from_path, recursive=recursive)
+            except FileNotFoundError:
+                # In some cases, there is no file in the original directory, so we just skip copying the file to the remote path
```

Comment:
can you add a debug log here mentioning that the file does not exist?

Reply:
yeah let's do that @pingsutw